pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -916,16 +916,29 @@ func (lrw *logicalReplicationWriterProcessor) dlq(
 
 type batchStats struct {
 	optimisticInsertConflicts int64
-	kvWriteFallbacks          int64
+	kvWriteTooOld             int64
+	kvWriteValueRefreshes     int64
 }
+
+func (b *batchStats) Add(o batchStats) {
+	b.optimisticInsertConflicts += o.optimisticInsertConflicts
+	b.kvWriteTooOld += o.kvWriteTooOld
+	b.kvWriteValueRefreshes += o.kvWriteValueRefreshes
+}
+
 type flushStats struct {
 	processed struct {
 		success, dlq, bytes int64
 	}
 	notProcessed struct {
 		count, bytes int64
 	}
-	optimisticInsertConflicts, kvWriteFallbacks int64
+
+	batchStats
+}
+
+func (b *flushStats) AddBatchStats(bs batchStats) {
```

Comment:
nit: i feel like this one line wrapper is a bit overkill for now. Could we just use `b.batchStats.Add()` and remove this?